src/components/__tests__/LibraryDetailPage-test.tsx

```diff
@@ -44,19 +52,26 @@ describe("LibraryDetailPage", () => {
   });
 
   it("should show the library information", () => {
-    let infoList = wrapper.find(".list-group");
-    expect(infoList.length).to.equal(1);
-    let infoItems = infoList.find("li");
-    expect(infoItems.length).to.equal(6);
-    infoItems.map((item) => {
+    let list = wrapper.find(".list-group");
+    expect(list.length).to.equal(2);
+
+    let basicInfoItems = list.at(0).find("li");
+    expect(basicInfoItems.length).to.equal(2);
+
+    let contactUrlItems = list.at(1).find("li");
+    expect(contactUrlItems.length).to.equal(4);
+
+    let allItems = Object.assign(basicInfoItems, contactUrlItems);
+    let libraryData = Object.assign(library.basic_info, library.urls_and_contact);
+    allItems.map((item: Enzyme.CommonWrapper<any, any, {}>) => {
       let key = item.find(".control-label").text();
-      let value = item.find(".form-control-static").text();
-      expect(`${library[key]}`).to.equal(value);
+      let value = item.find("span").text();
+      expect(`${libraryData[key]}`).to.equal(value);
```

Comment:
The key value is already a string so you don't need to extrapolate this variable.